### PR TITLE
Issue 5742 add youtube preview to vision page

### LINKED ISSUE #5742: add youtube videos to the /vision page
https://owocki.com/future-history-open-internet/
https://owocki.com/financial-system-people-talk/
https://owocki.com/open-source-sustainability-talk-ethereal/

i'd love to see these videos added to the /vision page on gitcoin.co

scope of this bounty is to come  up with a *responsive design* for the same.. and to code it up + PR it.

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description
Add youtube embed videos to /vision page

##### Refers/Fixes
Fixes #5742 

##### Testing
Go to `/vision` to view it.

### CODE REVIEW

**Comment by iamonuwa** [comment]
Screenshot from 2020-01-08 20-01-51

**Comment by iamonuwa** [comment]
Screenshot_20200108-200426_Chrome
Mobile

**Comment by L-KH** [comment]
As I read in bounty you should create a responsive front. I don't know how to PR, but you may add :

`.yourclass{max-width:100%;height:auto}img{vertical-align:middle}a{color:inherit}`
change yourclass to the video balize, this makes video responsive

File: app/retail/templates/vision.html [hunk 1]
```diff
@@ -238,6 +238,26 @@ <h3 class="mb-4 font-title-lg">{% trans "The Future Of Work" %}</h3>
...
+        </div>
+        <div class="container-fluid">
+          <div class="row justify-content-center">
+            {% for videoLink in videoLinks %}
+            <div class="col-12 col-md-4">
+              <iframe width="600" height="315" src="{{ videoLink }}" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"></iframe>
+            </div>
+            {% endfor %}
+        </div>
+        </div>
```
Comment (thelostone-mc) [inline app/retail/templates/vision.html]: duplicate div ?

**Comment by iamonuwa** [comment]
> @iamonuwa could we ensure the videos don't overflow the screen on mobile and videos have padding between them on mobile ?

Fixed this, thanks

**Comment by iamonuwa** [comment]
Mobile update
image

Desktop
image

**Comment by iamonuwa** [comment]
@thelostone-mc @octavioamu @danlipert @owocki any other issue raised?

**Comment by owocki** [comment]
LGTM but defer to the engineers for final say

**Pull request merged**